test/args_test.go

```diff
@@ -87,6 +87,19 @@ func TestAcc_Args_UserConfirmation(t *testing.T) {
 				"Are you sure you want to delete these resources (cannot be undone)? Only YES will be accepted.",
 			},
 		},
+		{
+			name:      "force",
+			extraArgs: []string{"--force"},
+			expectedLogs: []string{
+				"PROCEEDING WITH DELETION AND SKIPPING CONFIRMATION (FORCE)",
+				"TOTAL NUMBER OF RESOURCES THAT WOULD BE DELETED: 1",
+			},
+			unexpectedLogs: []string{
+				"STARTING TO DELETE RESOURCES",
+				"TOTAL NUMBER OF DELETED RESOURCES:",
+				"Are you sure you want to delete these resources (cannot be undone)? Only YES will be accepted.",
```

Comment:
Correct me if I'm wrong, but I believe this would always be included if there are resources to delete: https://github.com/christonog/awsrm/blob/b4d924cbd04cfdd117e5f24342bb992082e755cb/pkg/resource/resource.go#L66